roles/cobbler/templates/snippets/cephlab_rc_local

```diff
@@ -144,7 +147,7 @@ set +x
 echo -e "==================================\nInstructing Cobbler to run Ansible\n      Waiting for completion\n==================================" > /dev/console
 set -x
 # Run the post-install trigger a second time
-wget --timeout=1800 -t1 -O /dev/null "http://$http_server:$http_port/cblr/svc/op/trig/mode/post/system/$system_name" || true
+curl -s "http://$http_server:$http_port/cblr/svc/op/trig/mode/post/system/$system_name" -o /dev/null || true
```

Comment:
Is the lack of timeout here going to be a potential issue? Would using `/usr/bin/timeout` be helpful?